README.md

```diff
@@ -1,7 +1,9 @@
-# Go Databricks SQL Driver
+# Preview Go Databricks SQL Driver
```

Comment:
We should put the label at the end of the title.

Reply:
changed to beta and moved to the end.